next.clayui.com/content/docs/foundations/composing.md

```diff
@@ -3,4 +3,146 @@ title: "Composing"
 description: "Component composition philosophy"
 ---
 
-This is a stub. Help our community expand it.
\ No newline at end of file
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+- [Composition](#composition)
+- [Philosophy](#philosophy)
+    - [Low-level components](#low-level-components)
+	- [High-level components](#high-level-components)
+
+</div>
+</div>
+
+Clay follows some fundamentals for creating new components and as we distribute the components so that they can be used, the components follow some of the main React patterns: [composition](https://reactjs.org/docs/composition-vs-inheritance.html) or you can listen around for "bulding blocks".
```

Comment:
```suggestion
Clay follows some fundamentals for creating new components by following a common React pattern of 
[composition](https://reactjs.org/docs/composition-vs-inheritance.html).
```

Reply:
I don't think so. "listen around" doesn't make much sense for this context and "building blocks" is also similar to composition and low-level, so I think it'd be best to leave it out.

Reply:
Hmm, ok. Np.